### PR TITLE
Add missing changelog entry for new default Ddoc theme

### PR DESCRIPTION
**Pull request by jacob-carlborg**

### CODE REVIEW

**Comment by jacob-carlborg** [comment]
@MartinNowak ping.

**Comment by ZombineDev** [comment]
It would be cool if you could add a screenshot comparison.

**Comment by jacob-carlborg** [comment]
> It would be cool if you could add a screenshot comparison.

What is required to do that in terms of integrating with the build?

File: changelog/new_default_ddoc_theme.dd [hunk 1]
```diff
@@ -0,0 +1 @@
+A completely new default Ddoc theme has been created.
```
Comment (MartinNowak) [inline changelog/new_default_ddoc_theme.dd]: That's 3 adjectives in a row!
How about `The default Ddoc them has been overhauled.`?

**Review by MartinNowak — COMMENTED** [review]
Some more details would be nice.

When generating documentation using `-D`, dmd will now use a more modern looking theme, also see $(LINK2 https://github.com/dlang/dmd/pull/6173, dmd#6173).

**Comment by jacob-carlborg** [comment]
Thanks.

**Pull request merged**